t/v27_features.py

```diff
@@ -20,3 +20,92 @@
 assert(result['uri'] == '/issue_120/{}'), "expected result"
 print ('[+] OK')
 
+preset_method_location = BASE_URL + '/skipping_test'
+body = [
```

Comment:
We can use those test cases in two ways: send them in a batch (with the surrounding array) and send them one-by-one. So we'll verify both modes: batching and non-batching.

Reply:
Done.